src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -410,7 +438,144 @@ lemma tendsto_test_against_nn_of_le_const {μ : finite_measure α}
 tendsto_test_against_nn_filter_of_le_const
   (eventually_of_forall (λ n, eventually_of_forall (fs_le_const n))) (eventually_of_forall fs_lim)
 
-end finite_measure
+end finite_measure -- namespace
+
+end finite_measure_bounded_convergence -- section
+
+section finite_measure_convergence_by_bounded_continuous_functions
+/-! ### Weak convergence of finite measures with bounded continuous real-valued functions
+
+In this section we characterize the weak convergence of finite measures by the most common
+(defining) condition of convergence of integrals of all bounded continuous real-valued functions.
+-/
+
+namespace finite_measure
+
+variables {α : Type*} [measurable_space α] [topological_space α] [opens_measurable_space α]
+
+lemma integrable_of_bounded_continuous_to_nnreal
+  (μ : measure α) [is_finite_measure μ] (f : α →ᵇ ℝ≥0) :
+  integrable ((coe : ℝ≥0 → ℝ) ∘ ⇑f) (μ : measure α) :=
+begin
+  refine ⟨(nnreal.continuous_coe.comp f.continuous).measurable.ae_strongly_measurable, _⟩,
+  simp only [has_finite_integral, nnreal.nnnorm_eq],
+  exact lintegral_lt_top_of_bounded_continuous_to_nnreal _ f,
+end
+
+lemma integrable_of_bounded_continuous_to_nnreal' (μ : finite_measure α) (f : α →ᵇ ℝ≥0) :
+  integrable ((coe : ℝ≥0 → ℝ) ∘ ⇑f) (μ : measure α) :=
+begin
+  refine ⟨(nnreal.continuous_coe.comp f.continuous).measurable.ae_strongly_measurable, _⟩,
+  simp only [has_finite_integral, nnreal.nnnorm_eq],
+  exact lintegral_lt_top_of_bounded_continuous_to_nnreal _ f,
+end
+
+lemma integrable_of_bounded_continuous_to_real (μ : finite_measure α) (f : α →ᵇ ℝ) :
+  integrable ⇑f (μ : measure α) :=
```

Comment:
Two lemmas above, in a lemma you have replaced an assumption `(μ : finite_measure α)` with `(μ : measure α) [is_finite_measure μ]`. And indeed I think this is a better design when the `finite_measure` only appears in the conclusion of the lemma through its coercion to a measure: the new version of the lemma always applies when the old one used to apply, but we can also use it when we have a `measure`, not a `finite_measure`. For instance, they will automatically apply to (coercions of) probability measures.

I think the same holds in the current lemma, and also in most lemmas below.

Reply:
Now that you say it, it seems obvious: basically just more generally applicable results and less need for coercions, so simply better.

I changed this throughout the file (expect of course in results that rely on the topology instance on `finite_measure` or some other specific feature).

Thank you!